src/scss/custom/pages/_orders-return.scss

```diff
@@ -0,0 +1,33 @@
+.order-returns {
+  padding: 0.5rem;
+
+  .order-return {
+    padding: 0 1rem;
+    background: lighten($gray-100, 2%);
+    border-radius: 0.5rem;
```

Comment:
`.table-wrapper` already apply that background, padding and border radius

Looks like it's a duplicate